### DISCUSSION TITLE
Select language

### DISCUSSION THREAD

**Discussion by amadicus** [FossifyOrg/General-Discussion#102] [Ideas]
Hello.  Is it possible to change the language selection logic?  I currently have Chinese and English on my phone.  Now, when choosing a language, nothing happens (as I understand, the choice is tied to the localization of the phone), the program is in English.  But programs that have a choice of language and it is not associated with the localization of the phone (and the language files are in the program itself) change the language to any other.  Thank you!

**Comment by naveensingh** [comment]
@amadicus You can switch between supported languages using the `Settings => Language` option regardless of the system language.

**Reply by amadicus** [reply]
When I try to select a language in the settings, it throws me out of the settings.  The phone has two languages ​​- English and Chinese.

**Reply by naveensingh** [reply]
How did you download the app? Using F-Droid or Google Play?

Please share your device model and Android version.

**Reply by amadicus** [reply]
App from github
Nubia Z50Ultra, Android 13
